Tests/GitHubReactions.Tests.ps1

```diff
@@ -48,6 +50,8 @@ try
             It 'Should have the expected reaction content' {
                 $specificReactions.content | Should be $otherReactionType
```

Comment:
Please make sure you're using Pester 5 syntax here (even though we're not totally Pester 5 compliant yet...just don't want to add more debt), which means you have to be doing `-Be` instead of `Be`, etc...